index.jsx

```diff
@@ -459,19 +465,23 @@ class Controls extends Component {
       plusBtnContents,
       minusBtnContents,
       btnClass,
+      plusBtnClass,
+      minusBtnClass,
+      controlsClass,
       scale,
       minScale,
       maxScale
     } = this.props;
 
     const btnStyle = btnClass ? undefined : { width: 30, paddingTop: 5, marginBottom: 5 };
```

Comment:
btnStyle should be undefined only if both btnClass and plusBtnClass/minusBtnClass are undefined (depending on the button).